package.json

```diff
@@ -53,8 +53,5 @@
     "husky": "^8.0.3",
     "lint-staged": "^13.2.2",
     "prettier": "^2.8.8"
-  },
```

Comment:
Please do not delete a library that is used in application. Even there is not a dependency it is dangerous to delete a library.

Reply:
@DogukanGun I think Jonas accidentally added the dependency here. We only need it in `/graphql_server` but it is already in the `package.json` of `/graphql_server`

Reply:
that is why I deleted it in the root package.json